Tests/LinkTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LinkTests.swift
+//  Contentful
+//
+//  Created by JP Wright on 22/03/2017.
+//  Copyright © 2017 Contentful GmbH. All rights reserved.
+//
+
+@testable import Contentful
+import Foundation
+import XCTest
+import Nimble
+import DVR
+
+class LinkTests: XCTestCase {
+
+    func testGetLinkFromFieldValueThatIsAlreadyALink() {
+        let entry = Entry(localizedField: ["":["":""]])
+        let link = Link.entry(entry)
+        let newLink = Link.link(from: link)
+        expect(newLink).toNot(beNil())
```

Comment:
Can this test be more explicit? I.e. what about changing it from `.toNot(beNil())` to testing if the `newLink` sys.id equals the linked entry id, or so?